src/streamlink/plugins/pluzz.py

```diff
@@ -45,10 +45,10 @@ def _get_streams(self):
         # Retrieve geolocation data
         country_code = self.session.http.get(self.GEO_URL, schema=validate.Schema(
             validate.parse_json(),
-            {"reponse": {"geo_info": {
+            {"response": {"geo_info": {
                 "country_code": str
             }}},
-            validate.get(("reponse", "geo_info", "country_code"))
+            validate.get(("response", "geo_info", "country_code"))
```

Comment:
This is a data response from the server. Your automated tool for checking spelling errors would break the plugin.

Reply:
You can close, I just posted where you can correct errors